x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/classification_exploration/_classification_exploration.scss

```diff
@@ -22,16 +22,17 @@
 
 .mlDataFrameAnalyticsClassification__actualLabel {
   float: left;
-  width: 80px;
+  width: 8%;
   padding-top: $euiSize * 4;
 }
 
 /*
   Gives EuiDataGrid a min-width of 480px, otherwise the columns options will disappear if you hide all columns.
 */
 .mlDataFrameAnalyticsClassification__dataGridMinWidth {
+  float: left;
```

Comment:
I presume you don't need `float: left` for both `mlDataFrameAnalyticsClassification__actualLabel` and `mlDataFrameAnalyticsClassification__dataGridMinWidth`.
Have you tried using flex or simply inline-blocks?

Reply:
Testing this, `float: left` is needed on both for me.

Reply:
It was originally written with flex but there's a comment on it being changed to avoid using that so that it would support IE11.